project/src/com/google/daggerquery/dagger_query_textproto.bzl

```diff
@@ -25,7 +25,7 @@ def _dagger_query_textproto_impl(ctx):
     ctx.actions.run_shell(
         inputs = src_jars,
         outputs = [ctx.outputs.out],
-        command = "unzip -p {src_jar} binding_graph.textproto > {out}".format(
+        command = "unzip {src_jar} '*.textproto' -d .; zip {out} '*.textproto'".format(
```

Comment:
if it is common for a jar file to contain textprotos outside of this plugin, I think we should add a specific prefix or suffix to these binding graphs.

Reply:
Yes, thanks! I think `_graph.textproto` suffix will be pretty good.